src/frontend/shared-components/src/components/table/Table.stories.js

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+
+import Table from "./Table";
+
+export default {
+  title: "Table",
+  component: Table
+};
+
+const tableElements = [
+  { name: "BCeID", value: "bobross42" },
+  {
+    name: "Last Name",
+    value: "Ross"
+  },
+  { name: "First Name", value: "Bob" },
+  { name: "Email Address", value: "bob.ross@pbs.com" }
+];
+
+const table = {
+  header: "BCeID Info",
+  tableElements
+};
```

Comment:
can we move these into modules? in a function called `getTableTestData` or something? will clean it up, follow similar pattern as already set, and also fix this code climate issue :) 3 🐦 1 🥌

Reply:
3 birds one curling rock. Gotcha.